contract/gql_mutations/create_contract.py

```diff
@@ -0,0 +1,42 @@
+from contract.gql_mutations import ContractInputType
+from contract.gql_mutations.utils import update_or_create_contract
+from contract.models import Contract
+
+from .base_mutation import BaseContractMutation
+from core.utils import TimeUtils
+from django.contrib.auth.models import AnonymousUser
+from django.core.exceptions import ValidationError, PermissionDenied
+
+
+class CreateContractMutation(BaseContractMutation):
+    _mutation_class = "CreateContractMutation"
+
+    class Input(ContractInputType):
+        pass
+
+    @classmethod
+    def _validate_mutation(cls, user, **data):
+        if type(user) is AnonymousUser or not user.id:
+            raise ValidationError("mutation.authentication_required")
+        # if not user.has_perms(ContractConfig.gql_mutation_create_contract_perms):
+        #     raise PermissionDenied(_("unauthorized"))
+
+    @classmethod
+    def _mutate(cls, user, **data):
+        data['policy_holder'] = user.id_for_audit
+        data['status'] = Contract.STATUS_ENTERED  # TODO: Determine all possible statuses for contract ConfigEnum
+        data['validity_from'] = TimeUtils.now()
```

Comment:
I don't want to keep this beaviour(UUIDVersionnedModel), validityTo/ValidityFrom was repalced by ContractTo/ContractFrom that are for business logic (Contract for dec is from 1/12 to 31/12)

Reply:
But it is valid for DateUpdated/Created